source/connect.txt

```diff
@@ -141,6 +141,59 @@ the shell connected to your selected deployment.
 
    :mongosh:`Perform CRUD Operations in the MongoDB Shell </crud>`
 
+
+.. _vsce-connect-task-atlas-streams:
+
+Create a Connection to Atlas Stream Processing
```

Comment:
[suggestion]

Consider making this even more direct: "Connect to Atlas Stream Processing".

Reply:
Done.

Reply:
Can we add how to get the connection string?  

You can use MongoDB for VS Code to connect to 
[Atlas Stream Processing](https://www.mongodb.com/products/platform/atlas-stream-processing)
 by providing a Stream Processing Interface (SPI) connection string.

To obtain the connection string navigate to the desired Stream Processor Instance, click Connect and select "MonfoDB for VSCode"

![image](https://github.com/mongodb/docs-mongodb-vscode/assets/24432767/478709d1-eb7e-458f-83f1-64ddbac66fbe)

Reply:
Hi @RWaltersMA do you know ASP will be public? Currently, when I go to cloud.mongodb.com Stream Processing I can only request private preview access to the feature I added a link to the Stream processing docs as well. Should we add a note here that ASP is currently in private preview?